j2k/testData/fileOrElement/comments/commentsForConstructors.kt

```diff
@@ -8,8 +8,7 @@ internal class A// this is a primary constructor
 
     // this is a secondary constructor 2
     constructor(s: String) : this(s.length) {} // end of secondary constructor 2 body
-}// this is a secondary constructor 1
-// end of secondary constructor 1 body
```

Comment:
Not sure that it nice to silently drop the comment. 
It's better to place it after comment for declaration where default parameters inferred now.

Reply:
Thanks, will fix.